### PR TITLE
Fixes #11: Include repository cherry pick exists within

### LINKED ISSUE #11: Include repo in chery pick output
The cherry picks file doesn't specify which repositories the SHAs should be cherry picked in, so either splitting it up per repo or listing it somehow would be very useful.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@domcleal I tested this using a Katello config file and seemed to work OK for me, if you wouldn't mind testing it as well

**Comment by domcleal** [comment]
Yep, works well, thanks!

**Comment by ehelms** [comment]
I changed the formatting around to make doing repo by repo more obvious:

```
Cherry Picks for repository:
----------------------------------------------
10351 - 2015-05-28 21:01:55 UTC: [5b2db62a2a24e48eaa3b81b982ce066285c67e78] I can't re-promote a content view if it's in the last environment of a path

Cherry Picks for repository: katello
----------------------------------------------
10690 - 2015-06-05 00:01:50 UTC: [a0923faf416c1aaf31e240d4d3ae1186b6a81748] ActiveRecord::RecordInvalid: Validation failed: Title has already been taken
10948 - 2015-07-06 15:01:41 UTC: [d1da84ba45a80b0158eb1ad94431794376fcbb83] Unable to get unlimited-content-hosts value for host collection

Cherry Picks for repository: katello-installer
----------------------------------------------
10716 - 2015-06-29 15:02:26 UTC: [e4ef2c1cf8d625371f3abc4b41b820eabcc33b09] Capsule upgrade fails with unknown image type docker
11139 - 2015-07-21 02:01:43 UTC: [d4b15588253327b727698d4e0a7b81794fda09f3] 2.3 Upgrade Fails with Unable to Reload Puppet
```

File: lib/tool_belt/cherry_picker.rb [hunk 1]
```diff
@@ -76,5 +91,15 @@ def write_cherry_pick_log(picks, release)
...
     end
 
+    def log_entry(pick)
+      "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+    end
+
+    def find_repository(revision)
+      @release_environment.repo_names.find do |repo_name|
+        @release_environment.commit_in_repo?(repo_name, revision)
+      end
```
Comment (domcleal) [inline lib/tool_belt/cherry_picker.rb]: Perhaps add a `|| 'unknown'` or something here?

Comment (ehelms) [inline lib/tool_belt/cherry_picker.rb]: Updated the logic to output a section that is labeled unknown

Comment (domcleal) [inline lib/tool_belt/cherry_picker.rb]: Thanks!

**Pull request merged**